### ISSUE TITLE
No unused warning is shown for `type` operators

### ISSUE THREAD

**Issue by garyb** [purescript/purescript#3553]

**Comment by hdgarrood** [comment]
As in, if I write
```
import Prelude ((<$>))
```
and don't use `<$>` anywhere in that module, no warning will be generated?

**Comment by garyb** [comment]
Yes - although, I think that one will warn, there was a different issue about value operators in the past. The specific case I have in mind here is:

```
import Type.Row (type (+))
```

**Comment by hdgarrood** [comment]
:man_facepalming: yes, of course

**Comment by garyb** [comment]
Oh actually, I might still be slightly wrong about what's going on here - the case I encountered it was I added that to our `Prelude`, but all the modules that imported it elsewhere didn't warn about the existing imports becoming redundant. So the description probably isn't right here anyway.
